string task_id
string task_type
geometry_msgs/PoseStamped entity_position
geometry_msgs/PoseStamped target_position
